frontend/nginx/nginx.conf.template

```diff
@@ -100,10 +100,10 @@ server {
         proxy_set_header X-Forwarded-Host  $host;
 
         proxy_intercept_errors on;
-        error_page 503 = @custom_424;  # Redirect 503 errors when Plausible is down
+        error_page 502 503 = @custom_424;  # Redirect 502 and 503 errors when Plausible is down
```

Comment:
Should it include [504 errors](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/504)? It's very similar to those two.

Reply:
We only had 5 of 504s when there were thousands of others:
<img width="475" alt="Screenshot 2025-02-05 at 7 02 27 AM" src="https://github.com/user-attachments/assets/3e9b096f-b69f-473d-9657-9402c4d2e312" />
But I'll include them anyways, since they are in the same category.

By the way, this time, the problem with Plausible is reflected on the https://status.plausible.io/
<img width="889" alt="Screenshot 2025-02-05 at 7 06 56 AM" src="https://github.com/user-attachments/assets/d747dbb5-a025-41c9-83c7-23c3404818fc" />

Previously, when we had alarms, the status was green